src/CMakeLists.txt

```diff
@@ -107,7 +107,7 @@ set(libclang_source
 
 add_library(cppast ${detail_header} ${header} ${source} ${libclang_source})
 target_compile_features(cppast PUBLIC cxx_std_11)
-target_include_directories(cppast PRIVATE ../include SYSTEM INTERFACE ../include)
```

Comment:
I am not sure what was the idea here (and I am not a cmake expert) but with `PUBLIC` things install correctly for me.

Reply:
The effect is as follows: for cppast itself, the include headers are passed via -I, which enables warning in them. However, for users the include headers are passed via -isystem, which disables warnings.

What is the issue if it is kept as-is?

Reply:
If I remember correctly, they were not included correctly in the install target like this.

Reply:
Let me check what happens without this.

Reply:
Ok. This is actually not needed. Apparently I changed something else when I modified this and that made things work.